packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.html

```diff
@@ -0,0 +1,98 @@
+<div data-ui-view>
+    <div class="octavia-load-balancers">
+        <oui-header
+            data-heading="{{ :: 'octavia_load_balancers_title' | translate }}"
+        ></oui-header>
+
+        <div data-ovh-alert="octavia.alerts.global"></div>
+        <oui-datagrid
+            class="table-striped"
+            data-page-size="{{ $ctrl.paginationSize }}"
+            data-rows="$ctrl.resources"
+            data-criteria="$ctrl.criteria"
+            data-on-page-change="$ctrl.onPageChange($pagination)"
+            data-on-criteria-change="$ctrl.onCriteriaChange($criteria)"
+            data-on-sort-change="$ctrl.onSortChange($sort)"
+            data-customizable
+        >
+            <oui-datagrid-topbar>
+                <oui-button
+                    data-ng-click="$ctrl.createLoadBalancer()"
+                    variant="secondary"
+                    icon-left="oui-icon-plus"
+                >
+                    <span
+                        data-translate="octavia_load_balancers_add_button"
+                    ></span>
+                </oui-button>
```

Comment:
```suggestion
                <oui-button
                    data-ng-click="$ctrl.goTocreate()"
                    variant="secondary"
                    icon-left="oui-icon-plus"
                    data-at-internet-click="{name: $ctrl.trackingCreateLoadBalancer}"
                >
                    <span
                        data-translate="octavia_load_balancers_add_button"
                    ></span>
                </oui-button>
```

Maybe you can use track internet directive here so no need to call an intermediate controller method to make the redirection

Reply:
This is not possible. When I tried to do it, I got an error:

Error: [$compile:multidir] Multiple directives [atInternetClick (module: ngAtInternet), ouiButton] asking for new/isolated scope on: <oui-button data-ng-click="$ctrl.goToCreatePage()" variant="secondary" icon-left="oui-icon-plus" data-at-internet-click="{name: $ctrl.trackingCreateLoadBalancer}">